cvat/apps/dataset_manager/bindings.py

```diff
@@ -490,14 +530,36 @@ def _make_image(i, **kwargs):
                 'path': frame_data.name + ext,
                 'size': (frame_data.height, frame_data.width),
             }
-            if include_images:
+
+            if dimension == DimensionType.DIM_3D:
+                dm_image = _make_image(frame_data.id, **image_args)
+            elif include_images:
                 dm_image = _make_image(frame_data.idx, **image_args)
             else:
                 dm_image = Image(**image_args)
             dm_anno = self._read_cvat_anno(frame_data, task_data)
-            dm_item = datumaro.DatasetItem(id=osp.splitext(frame_data.name)[0],
-                annotations=dm_anno, image=dm_image,
-                attributes={'frame': frame_data.frame})
+
+            if dimension == DimensionType.DIM_2D:
+                dm_item = datumaro.DatasetItem(id=osp.splitext(frame_data.name)[0],
+                                               annotations=dm_anno, image=dm_image,
+                                               attributes={'frame': frame_data.frame})
+            elif dimension == DimensionType.DIM_3D:
+                attributes = {'frame': frame_data.frame}
+                if format_type == "sly_pointcloud":
+                    attributes["name"] = self._user["name"]
+                    attributes["createdAt"] = self._user["createdAt"]
+                    attributes["updatedAt"] = self._user["updatedAt"]
+                    attributes["labels"] = []
+                    index = 0
+                    for _, label in task_data.meta['task']['labels']:
+                        attributes["labels"].append({"label_id": index, "name": label["name"], "color": label["color"]})
+                        attributes["track_id"] = -1
+                        index += 1
+
+                dm_item = datumaro.DatasetItem(id=osp.split(frame_data.name)[-1].split('.')[0], image=None,
```

Comment:
```suggestion
                dm_item = datumaro.DatasetItem(id=osp.splitext(frame_data.name)[0],
```

Reply:
Hi @zhiltsov-max , the **osp.split(frame_data.name)[-1].split('.')[0]** returns the entire file path and not just the fileName.
We can leave it as above or change to : 
id=osp.splitext(osp.split(frame_data.name)[-1])[0]

Reply:
Why do you need to return only basename?

Reply:
hi @zhiltsov-max,
As per the suggest changes we are getting the folder structure as shown below, **"test_canvas3d/pointcloud/"** is  nested inside **ann, pointcloud and related images** folders before the actual data.

![image](https://user-images.githubusercontent.com/25260417/125949868-0e31de5b-c46b-4fea-a82f-d7ea40696fb5.png)

Hence we need only require the basename

Reply:
CVAT preserves relative image paths on export, so such behavoiur is probably expected. But it can be a problem of the specific test.

Reply:
@zhiltsov-max, If we make the suggested change _**_id=osp.splitext(frame_data.name)[0]_**_ then the folder structure downloaded on click of dump annotations manually is as above which is incorrect as per SOW and not compatible with supervisely.